rotkehlchen/chain/ethereum/utils.py

```diff
@@ -219,3 +220,14 @@ def try_download_ens_avatar(
     avatars_dir.mkdir(exist_ok=True)  # Ensure that the avatars directory exists
     with open(avatars_dir / f'{ens_name}.png', 'wb') as f:
         f.write(avatar)
+
+
+def has_tracked_addresses(
+        blockchain: SupportedBlockchain,
+        database: 'DBHandler',
+) -> bool:
+    """Returns true if we are tracking any address for the blockchain"""
```

Comment:
But in reality this does not need a docstring. I would remove it.

```suggestion
    """Returns true if we are tracking any address for the given blockchain"""
```